Automattic/jetpack
# Conversation: remove unused participants json file

This PR removes the `participants.json` file not used anymore by the editor app.

Fixes #

#### Changes proposed in this Pull Request:

* Conversation: remove unused participants json file

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






Create / Edit a Conversation/Dialogue blocks composition. Everything should be ok.

#### Proposed changelog entry for your changes:


*
